visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts

```diff
@@ -187,7 +202,7 @@ export class CodeMapMouseEventService
 
 	private onLeftClick() {
 		this.threeSceneService.clearSelection()
-		if (this.intersectedBuilding) {
+		if (this.intersectedBuilding && !this.hasMouseMoved(this.mouseOnLastClick)) {
```

Comment:
Why is this required?

Reply:
> We keep track of the mouse position to prevent the map from opening a link to a building or the sidebar when releasing the left-button on a building after moving the map

I should've read your wonderful PR description. Thanks for that.
Nonetheless: My question here might come up later again, when someone looks at the code. Please add a small comment over the condition, explaining why this is necessary.